src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="month month--start-day-fri month--month-length-31">
+        <div class="month--day-1"></div>
+        <div class="month--day-2"></div>
+        <div class="month--day-3"></div>
+        <div class="month--day-4"></div>
+        <div class="month--day-5"></div>
+        <div class="month--day-6"></div>
+        <div class="month--day-7"></div>
```

Comment:
These should be the elements of `month` block, something like `month__day`. And they should not have any modifiers.